### PR TITLE
Add auxiliary diagnostic format and run `check` and `test` completely when running `rome ci`

### PR DESCRIPTION
**Pull request by sebmck**
This PR does two things:

**Run `check` and `test` completely when running `rome ci`**

Before if `rome check` failed we would never run `rome test`. Now we always run them and concat any diagnostics printers at the end.

**Add auxiliary diagnostic format**

This is a new flag that gets derived from the env vars. When set we output another format alongside diagnostics. I've implemented support for GitHub Actions so now when running `rome test` or `rome check` inside of it, PRs should automatically get inline errors.

### CODE REVIEW

**Comment by sebmck** [comment]
We now have colors in CI:

Screen Shot 2020-07-09 at 1 45 00 AM

And inline errors on the "Files Changed" tab:

Screen Shot 2020-07-09 at 2 12 05 AM

File: packages/@romefrontend/compiler/lint/rules/js/noMultipleSpacesInRegularExpressionLiterals.test.md [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
   ℹ Recommended fix
 
+  - foo ·bar
   + fo {3}ar
```
Comment (yassere) [inline packages/@romefrontend/compiler/lint/rules/js/noMultipleSpacesInRegularExpressionLiterals.test.md:24]: This isn't related to this PR, but this looks like a bug.

Comment (sebmck) [reply packages/@romefrontend/compiler/lint/rules/js/noMultipleSpacesInRegularExpressionLiterals.test.md:24]: @yassere Want to open an issue to track?

**Comment by sebmck** [comment]
I'm going to merge this once it's green. I'm unhappy that there are so many changes in this PR and that the scope increased. In the process of making the CI command better I noticed that other things were awful and tried to fix those too which ended up taking MUCH longer than I anticipated.

**Comment by sebmck** [comment]
🙏

**Pull request merged**